docs/Release_notes.md

```diff
@@ -15,6 +15,9 @@ Enhancements in this release:
 Known Issues:
 - No known issues at this time.
 
+Breaking Changes:
+- Users should note that due to the switch to J2735 2020, some fields in outputted TIMs will be different. Any programs relying on data from the ODE must be modified to accommodate these updated outputs.
```

Comment:
Lets specify the relevant fields here or a link to a place noting the differences for convenience. Not everyone is an expert of the J2735.

Reply:
Good call! The relevant fields & their new names have been added to this section.

Reply:
Awesome